### ISSUE TITLE
Improve leaving insert mode on load

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#5605]
We could leave insert mode when load finished only if no editable element is currently focused.

I tried this but ran into some trouble because the check is asynchronous, so this requires a bit more thought first. Reverted in 5082331d46a26f03e2adeecf94d4a61b6392a980.

**Comment by jpalus** [comment]
Not sure if that's about the same issue that I'm having, I would appreciate confirming:

when I open page I immediately go to input field (manually with gi) and start typing, however most likely upon page load complete event qutebrowser leaves insert mode and starts interpreting keystrokes as commands. That's actually a bit of a security risk as it occurred to me few times already that I entered login page, hit KeePassXC global combination for auto-typing credentials and my password ended up in google.

**Comment by The-Compiler** [comment]
@jpalus Yep, indeed. Unfortunately it's not possible to leave insert mode when loading started due to a Qt bug (also see #5931). You can set `input.insert_mode.leave_on_load` to not leave insert mode at all, but that might need getting some used to because e.g. after a google/ddg/... search you'll still be in insert mode.
